packages/dx-react-scheduler/src/plugins/day-view.tsx

```diff
@@ -76,30 +75,21 @@ class DayViewBase extends React.PureComponent<VerticalViewProps, ViewState> {
     dayScaleLayoutComponent: 'DayScaleLayout',
     dayScaleCellComponent: 'DayScaleCell',
     dayScaleRowComponent: 'DayScaleRow',
+    timeTableContainerComponent: 'TimeTableContainer',
     timeTableLayoutComponent: 'TimeTableLayout',
     timeTableCellComponent: 'TimeTableCell',
     timeTableRowComponent: 'TimeTableRow',
   };
 
-  timeTableElementComputed = () => this.timeTable;
-  layoutElementComputed = () => this.layout;
-  layoutHeaderElementComputed = () => this.layoutHeader;
-
-  layoutHeaderElement = memoize(viewName => (getters) => {
+  scrollingStrategy = memoize((viewName, scrollingStrategy) => (getters) => {
```

Comment:
`scrollingStrategy` and `timeTableElementsMeta` should be extracted outside the class

Reply:
Or is it connected with that problem with multiple views of the same type?

Reply:
Removed from all view plugins